neqo-transport/src/lib.rs

```diff
@@ -70,8 +70,8 @@ const ERROR_AEAD_LIMIT_REACHED: TransportError = 15;
 #[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq)]
 pub enum Error {
     NoError,
-    // Each time tihe error is return a different parameter is supply.
-    // This will be use to distinguish each occurance of this error.
+    // Each time this error is returned a different parameter is supplied.
+    // This will be used to distinguish each occurance of this error.
```

Comment:
Note that I only fixed the typos in this comment. I am not sure the comment itself is still applicable.

(Also let me know if you would prefer separate pull requests for these non-code drive-by changes.)